### PR TITLE
Make YouTube videos work in the hosted video template

### PR DESCRIPTION
**Pull request by lps88**
This is a WIP, just to get it playing at the moment!
## Screenshots

http://localhost:9000/advertiser-content/destination-canada-food-busker-in-canada/dueling-bagels
image
## Request for comment

@guardian/labs-beta

### CODE REVIEW

File: commercial/app/views/hosted/guardianHostedVideo.scala.html [hunk 1]
```diff
@@ -45,20 +45,24 @@
...
-                    poster="@{page.video.posterUrl}"
-                    class="vjs-hosted__video hosted__video gu-media--video vjs vjs-paused vjs-controls-enabled vjs_video_1-dimensions vjs-user-active">
-                        <source type="video/mp4" src="@{page.video.srcUrlMp4}">
-                        <source type="video/webm" src="@{page.video.srcUrlWebm}">
-                        <source type="video/ogg" src="@{page.video.srcUrlOgg}">
-                        <source type="video/m3u8" src="@{page.video.srcM3u8}">
-                    </video>
+                    @if(page.video.youTubeHtml.isDefined){
+                        @Html(page.video.youTubeHtml.get)
+                    } else {
```
Comment (kelvin-chappell) [inline commercial/app/views/hosted/guardianHostedVideo.scala.html]: Might be better as:
```
@page.video.youTubeHtml map { youTubeHtml =>
  @Html(youTubeHtml)
} getOrElse {
```

**Review by kelvin-chappell — APPROVED** [review]
👍

**Comment by paperboyo** [comment]
Can we switch off the display of youtube's title (lhs, top) and their own icons (rhs, top)? The tile clashes with our design badly here and the font is foreign. And their icons... are theirs...

This pertains to our lovely documentary page as well, of course: https://www.theguardian.com/world/ng-interactive/2016/sep/16/gun-nation-a-journey-to-the-heart-of-americas-gun-culture-video

@akash1810?

**Comment by akash1810** [comment]
> Can we switch off the display of youtube's title (lhs, top) and their own icons (rhs, top)? 

Yep, via [showinfo](https://developers.google.com/youtube/player_parameters#showinfo).

This PR is just to get the base template in, its not the final final implementation iirc.

**Comment by paperboyo** [comment]
> Yep, via showinfo.

🎆 Thx! 🎆

**Pull request merged**